src/objective-c/GRPCClient/GRPCCall.m

```diff
@@ -139,8 +146,10 @@ - (void)finishWithError:(NSError *)errorOrNil {
   _self = nil;
```

Comment:
I don't think it is possible to pick a single default value that will make sense in all circumstances - sufficiently long that calls that will eventually succeed aren't needlessly pruned (think space probes that are light-days away) yet sufficiently short that RAM doesn't fill up with calls that are uselessly old. Best to document that calls in-flight retain themselves until they either complete or are cancelled, or give up on the retain idea, and just add a dealloc that cancels if needed so that it's the client programmer's responsibility to keep references to calls-in-flight that the client cares about.

Maybe set the default to 1 minute,  with a header comment documenting that the client programmer may need to change this.

See http://www.ibiblio.org/harris/500milemail.html for more information.